tracing-subscriber/Cargo.toml

```
@@ -50,7 +50,7 @@ serde = { version = "1.0", optional = true }
 tracing-serde = { path = "../tracing-serde", version = "0.1.1", optional = true }
 
 # opt-in deps
-parking_lot = { version = ">= 0.7, < 0.10", optional = true }
+parking_lot = { version = ">= 0.7, < 0.11", optional = true }
```

should probably be 
```suggestion
parking_lot = { version = ">= 0.7, <= 0.11", optional = true }
```
as we also want to allow `0.11`

---

Whoops!